content/modules/r/pages/paste-and-paste0.adoc

```diff
@@ -0,0 +1,57 @@
+= `paste` and `paste0`
+
+`paste` is a useful function to "concatenate vectors after converting to character."
```

Comment:
`paste` is a function that converts vector elements to character and then concatenates (links) them.